From c320799c941d42a9b7e351805359694ea3b36d2d Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Sun, 5 Apr 2015 09:08:38 -0700
Subject: [PATCH] nls.m4: Take it from gettext-0.15

It otherwise expects gettext to provide it. Lets go independent

Signed-off-by: Khem Raj <raj.khem@gmail.com>

Upstream-Status: Pending
---
 m4/nls.m4 | 30 ++++++++++++++++++++++++++++++
 1 file changed, 30 insertions(+)
 create mode 100644 m4/nls.m4

diff --git a/m4/nls.m4 b/m4/nls.m4
new file mode 100644
index 0000000..2c8fe69
--- /dev/null
+++ b/m4/nls.m4
@@ -0,0 +1,30 @@
+dnl Copyright (C) 1995-2003, 2005-2006 Free Software Foundation, Inc.
+dnl This file is free software; the Free Software Foundation
+dnl gives unlimited permission to copy and/or distribute it,
+dnl with or without modifications, as long as this notice is preserved.
+dnl
+dnl This file can can be used in projects which are not available under
+dnl the GNU General Public License or the GNU Library General Public
+dnl License but which still want to provide support for the GNU gettext
+dnl functionality.
+dnl Please note that the actual code of the GNU gettext library is covered
+dnl by the GNU Library General Public License, and the rest of the GNU
+dnl gettext package package is covered by the GNU General Public License.
+dnl They are not in the public domain.
+
+dnl Authors:
+dnl Ulrich Drepper drepper@cygnus.com, 1995-2000.
+dnl Bruno Haible haible@clisp.cons.org, 2000-2003.
+
+AC_PREREQ(2.50)
+
+AC_DEFUN([AM_NLS],
+[
+AC_MSG_CHECKING([whether NLS is requested])
+dnl Default is enabled NLS
+AC_ARG_ENABLE(nls,
+[ --disable-nls do not use Native Language Support],
+USE_NLS=$enableval, USE_NLS=yes)
+AC_MSG_RESULT($USE_NLS)
+AC_SUBST(USE_NLS)
+])
-- 
2.1.4

